.azure-pipelines/security-tools.yml

```diff
@@ -53,14 +53,22 @@ jobs:
       outputFormat: sarif
       scanFolder: SecurityTmp
       suppressionsFile: tools/SecurityTools/CredScanSuppressions.json
-      debugMode: false
-      folderSuppression: false
```

Comment:
Should we keep these two parameters? @dingmeng-xue

Reply:
No. they are default configuration or not supported.